.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Documentation
-about: Describe this issue template's purpose here.
+about: Describe the missing/wrong information in the documentation.
```

Comment:
Seems like you've rebased some commits from `main` into your branch.

Reply:
I have used the tooling in github to rebase pending commits. I'm actually confused that this commits are shown as different. I don't know how to workaround this.

Reply:
seems like it was a UI issue in Github, I made a minor change in the latest commit message (added a space) and force-pushed. now it seems to display correctly